File: Codigo/Semana_02/Reto_01/Fases_Obesidad.vb
' Este codigo ha sido generado por el modulo psexport 20180802-w32 de PSeInt.
' Es posible que el codigo generado no sea completamente correcto. Si encuentra
' errores por favor reportelos en el foro (http://pseint.sourceforge.net).

Module FASES_OBESIDAD

	Sub Main()
		Dim hijo As Double
		Dim madre As Integer
		Dim padre As Integer
		Console.WriteLine("Cuál es el peso del padre en kg: ")
		padre = Integer.Parse(Console.ReadLine())
		madre = 2*padre+4
		hijo = (madre+padre)/5
		Console.WriteLine(padre)
		Console.WriteLine(madre)
		Console.WriteLine(hijo)
		If hijo>=0 And hijo<=20 Then
			Console.WriteLine("uno")
		Else
			If hijo>=21 And hijo<=40 Then
				Console.WriteLine("dos")
			Else
				If hijo>=41 And hijo<=80 Then
					Console.WriteLine("tres")
				Else
					If hijo>80 Then
						Console.WriteLine("cuatro")
					End If
				End If
			End If
		End If
	End Sub

End Module

File: Codigo/Ciclo_01/Semana_02/Reto_01/Fases_Obesidad.py
print("Cual es el peso del padre en kg: ")
padre = int(input())
madre = int(2*padre+4)
hijo = int((madre+padre)/5)
print(padre, madre, hijo)
if hijo>=0 and hijo<=20:
		print("uno")
else:
		if hijo>=21 and hijo<=40:
			print("dos")
		else:
			if hijo>=41 and hijo<=80:
				print("tres")
			else:
				if hijo>80:
					print("cuatro")

